community/weechat/APKBUILD

```diff
@@ -1,14 +1,14 @@
 # Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
 pkgname=weechat
-pkgver=2.5
-pkgrel=2
+pkgver=2.6
+pkgrel=0
 pkgdesc="A fast, light, extensible ncurses-based chat client"
 url="https://weechat.org"
 arch="all"
 options="!check" # test suite runs "sudo make install"
 license="GPL-3.0-or-later"
 depends_dev="cmake libintl ncurses-dev gnutls-dev libgcrypt-dev curl-dev
-	aspell-dev lua-dev perl-dev python2-dev ruby-dev zlib-dev"
+	aspell-dev lua-dev perl-dev pkgconf python3-dev ruby-dev zlib-dev"
```

Comment:
`pkgconf` is not necessary

Reply:
- the `if..fi` [section](https://github.com/alpinelinux/aports/pull/11189/files#diff-149378f18b436ee1f7d4b7905430eea9R43) (lines 43..45) can be removed since python2 is no longer a fallback.
- you can also remove `builddir="$srcdir/$pkgname-$pkgver"` and `cd "$builddir"` at lines 25 and 60.

Reply:
Removed. Thank you.